Jenkinsfile

```diff
@@ -146,17 +145,25 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
             sh 'make defconfig'
         }
         stage('Build') {
-            // Attempt to build quickly and reliably
-            try {
-                sh "make -j4 V=s ${params.ALL_PACKAGES ? 'IGNORE_ERRORS=m' : ''}"
-            } catch (hudson.AbortException err) {
-                // TODO BUG JENKINS-28822
-                if(err.getMessage().contains('script returned exit code 143')) {
-                    throw err
+            // Add opkg signing key
+            withCredentials([
+                [$class: 'FileBinding', credentialsId: 'opkg-build-private-key', variable: 'PRIVATE_KEY'],
+                [$class: 'FileBinding', credentialsId: 'opkg-build-public-key', variable: 'PUBLIC_KEY'],
+            ]){
+                sh "ln -s ${env.PRIVATE_KEY} ${WORKSPACE}/key-build"
```

Comment:
Sounds like we have no choice if the key copying doesn't follow symlinks. Just make sure you wrap it in try/finally so that its always deleted